cmd/plugin/cli/common.go

```diff
@@ -82,3 +85,18 @@ func valueOrDefault(value string, def string) string {
 func stringToBase64(value string) string {
 	return base64.StdEncoding.EncodeToString([]byte(value))
 }
+
+func versionSupportsPortal(existingOperatorVersion string) error {
+	// TODO: remove develop check after 2.6 release
```

Comment:
What do you think about moving 2.5.x to a separate branch, and we don't need this check on main?